File: latihan2/SivitasAkademik.java
package DPBO.latihan2;

class SivitasAkademik extends Human{
    private String asal_universitas;
    private String email_edu;

    public SivitasAkademik extend Human(String asal_universitas, String email_edu){
        super(asal_universitas, email_edu);
        this.asal_universitas = asal_universitas;
        this.email_edu = email_edu;
    }

    public String getAsalUniversitas(){
        return asal_universitas;
    }

    public void setAsalUniversitas(String asal_universitas){
        this.asal_universitas = asal_universitas;
    }

    public String getEmailEdu(){
        return email_edu;
    }

    public void setEmailEdu(String email_edu){
        this.email_edu = email_edu;
    }
}

File: C++/SivitasAkademik.cpp
#include <iostream>
#include <string>

using namespace std;

//subclass
class SivitasAkademik : public Human {
   private:
    string asal_universitas;
    string email;

   public:
    //constructor
    SivitasAkademik(string NIK, string nama, char jenis_kelamin, string asal_universitas, string email) : Human(NIK, nama, jenis_kelamin) {
        this->asal_universitas = asal_universitas;
        this->email = email;
    }

    //constructor
    SivitasAkademik() : Human() {
        this->asal_universitas = "";
        this->email = "";
    }

    //set asal universitas
    void setAsalUniversitas(string asal_universitas){
        this->asal_universitas = asal_universitas;
    }

    //get asal universitas
    string getAsalUniversitas() {
        return asal_universitas;
    }

    //set email
    void setEmail(string email){
        this->email = email;
    }

    //get email
    string getEmailEdu() {
        return email;
    }
};
